src/decorator.ts

```diff
@@ -66,17 +72,20 @@ export class Decorator {
     return !!(this.activeEditor?.selections.find((s) => !(range.end.line < s.start.line || range.start.line > s.end.line)));
   }
 
-  getTogglableSymmetricRanges(documentText: string, regex: RegExp, length: number): Range[] {
+  getTogglableSymmetricRanges(documentText: string, regex: RegExp): Range[] {
     if (!this.activeEditor) return [];
 
     let match;
     const ranges = [];
     while ((match = regex.exec(documentText))) {
       const group = match[0];
 
+      const startGroup = match[1] || [];
```

Comment:
This is clever, love it!

Reply:
I just realized that although this code works because only the length property is used, I should have actually used 
```ts
const startGroup = match[1] || '';
const endGroup = match[match.length - 1] || '';
```
, as match is effectively an array of strings.